packages/tabs/src/Tabs/Tabs.tsx

```diff
@@ -58,10 +55,11 @@ const Tabs = (props: AccessibleTabsProps) => {
     children,
     className,
     darkMode: darkModeProp,
+    forceRenderAllTabPanels = false,
     inlineChildren,
     selected: controlledSelected,
     setSelected: setControlledSelected,
-    forceRenderAllTabPanels = false,
+    size = 'default',
```

Comment:
Would `baseFontSize` always be overridden since we're setting the default `size` to `default`?